### ISSUE TITLE
Feature 5: Facebook Help

### ISSUE THREAD

**Issue by Tashbezaleh** [Tashbezaleh/Tashbezaleh#7]
Feature Name: Facebook Help
Description: If a user didn't find the solution, he can ask it in facebook. (to decide whether facebook authentication is something we want – not too complex for users)
Requires: understanding facebook api, html design.
Due: Nice to have
Status: Nothing
Time Estimation: 12 hours

**Comment by Tashbezaleh** [comment]
Canceled. (Unnecessary)
